test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
same question here: Should I remove the test folder in all the branches? Or should I wait until I merge?